vex/otp-28.openvex.json

```diff
@@ -2,30 +2,30 @@
   "@context": "https://openvex.dev/ns/v0.2.0",
   "@id": "https://openvex.dev/docs/public/otp/vex-otp-28",
   "author": "vexctl",
-  "timestamp": "2025-08-20T12:53:19.493909+02:00",
-  "last_updated": "2025-08-20T12:53:19.824656672+02:00",
+  "timestamp": "2025-08-21T10:27:07.765086+02:00",
+  "last_updated": "2025-08-21T10:27:08.083410653+02:00",
   "version": 9,
   "statements": [
     {
       "vulnerability": {
         "name": "OSV-2025-300"
       },
-      "timestamp": "2025-08-20T12:53:19.712938265+02:00",
+      "timestamp": "2025-08-21T10:27:07.978996997+02:00",
       "products": [
         {
-          "@id": "pkg:otp/erlang@28.0"
+          "@id": "pkg:github/otp/erlang@28.0"
```

Comment:
When using `github` purls, the version is a commit or tag: https://github.com/package-url/purl-spec/blob/414fef487025046691af67f70dfa8677139df92d/types/github-definition.json#L23

```suggestion
          "@id": "pkg:github/otp/erlang@OTP-28.0"
```

Reply:
Good catch! It should be done in the latest changes.
Thanks!